woocommerce/woocommerce
# Fix new category name field error

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





An issue has been identified in the process of creating a new category. Specifically, after a new category is created, the input field retains the new category name instead of resetting.

![Screen Capture on 2023-08-23 at 16-36-17](https://github.com/woocommerce/woocommerce/assets/1314156/58b889d3-9cab-4c70-9ee7-3f818f4c137d)

This PR introduces code changes to address this problem.

![Screen Capture on 2023-08-23 at 16-37-23](https://github.com/woocommerce/woocommerce/assets/1314156/7eaab36e-a790-49d4-b849-cff47a0a7bd4)

Closes #39825.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a global attribute and a few terms under Products > Attributes. 
2. Enable the product blocks editor under WooCommerce -> Settings -> Advanced -> Features.
3. Navigate to the `Add product` page.
4. Under `Organization`, click the dropdown menu below `Categories`.
5. Input a new category name and click `+ Create "[category-name]"`.
6. A modal for creating the category will appear. Click `Save`.
7. Reopen the dropdown menu and verify that the new category name is no longer present in the input field.
8. Test the addition of attributes to ensure normal functionality.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
